packages/keystone/prepareKeystoneApp.js

```
@@ -4,16 +4,31 @@ const express = require('express')
 const conf = require('@open-condo/config')
 const { getRequestIp } = require('@open-condo/miniapp-utils/helpers/proxying')
```

ESLint

[error] There should be no empty line within import group

(import/order)